package.json

```
@@ -83,41 +94,48 @@
 		"webpack-merge": "4.1.2"
 	},
 	"files": [
+		"@types/",
 		"dist/",
 		"lib/",
+		"configs/",
 		"maintenance/",
 		"src/",
 		".babelrc",
-		".npmrc",
-		"jsconfig.json",
 		"package.json",
-		"package-lock.json",
-		"README.md",
-		"webpack.config.js"
+		"webpack.config.js",
+		"rollup.config.js",
+		"tslint.json",
+		"tslint.app.json"
 	],
 	"main": "./dist/ad-engine.js",
+	"module": "dist/index.es5.js",
+	"types": "dist/types/index.d.ts",
 	"nyc": {
 		"report-dir": "./spec/build/",
 		"reporter": [
 			"cobertura"
 		]
 	},
 	"scripts": {
-		"build": "webpack --progress --colors --hide-modules && node ./maintenance/build-bundle.js",
-		"ci-check": "npm run test && npm run lint",
 		"dev": "npm run serve",
+		"serve": "./node_modules/webpack-dev-server/bin/webpack-dev-server.js --progress --colors --inline --open --watch --content-base=./examples",
+		"prebuild": "rimraf dist",
```

Why are we using external dependency instead of simple
```suggestion
		"prebuild": "rm -rf dist",
```